src/actions/targetActions.js

```diff
@@ -10,7 +10,7 @@ export function createTargetSuccess(target){
 export const createTarget = (target) => {
   return dispatch => {
     return targetClient.createTarget(target).then(data => {
-      dispatch(createAlert("SideBarContainer", "Target created successfully", "success"));
+      dispatch(createAlert("SideBarContainer", "Target successfully created", "success"));
```

Comment:
I looked at the code of the `createAlert` function and I think it's a good idea to refactor those "SideBarContainer" and "success" flags to constants.

I'd declare a goals object with keys, for example:
```
const goals = {
  sidebar: 'SidebarContainer',
  ...etc
}
```
in a constants file, then import it and use it like `constants.sidebar`.

Also, remember that when writing JS, single quotes `'` are preferred.

I suggest you do the same with the alert types, so you don't have to use explicit strings all over the code.